src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/NuGetBrokeredServices.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ServiceHub.Framework;
+
```

Comment:
`#nullable enable` here would let callers know that the field below is never null.